### DISCUSSION TITLE
What is the best model of TTS for voice clone?

### DISCUSSION THREAD

**Discussion by LeiJiang1** [coqui-ai/TTS#2858] [General Q&A]
I tried different pretrained models of coqui-ai, given a sample test and voice, but none of models work well for voice cloning. Since there are many pre-trained models in coqui-ai, any ideas about which pre-trained models work best for voice clone? Do I have to do fine-tuning if I expect a good voice clone model?

**Comment by anni-anni** [comment]
I've tried various free models, some free trial versions of paid models for my video content voiceover. 
Finally settled on the Wavel [AI voice cloning](https://wavel.ai/studio/ai-voice-cloner/) software. Although features might be similar with other available AI models, it has worked way better than the others in my case. I've been using it for the past 7 months now and the dubbing (my voice cloned into other languages) sounds very natural I have to say.

**Comment by SlashHarken** [comment]
You generally get better results from fine-tuning assuming you have a large enough dataset. I would recommend using VITS if you have enough VRAM to save yourself the trouble of training a vocoder separately.

**Reply by rtruszkowski** [reply]
why not yourtts? I thought yourtts was objectively better and faster than vits?

**Comment by m-pektas** [comment]
I tried end2end multilingual models without finetuning in tts, the best one was xtts_v2 but it wasn't good enough for me. Do you have a better suggestion about voice cloning without fine-tuning in tts or a different repository?

**Comment by erogol** [comment]
https://huggingface.co/spaces/coqui/xtts
